src/demos.py

```diff
@@ -191,19 +190,25 @@ def pipeline_demo():
             )
             force_execution = choice == "y" or choice == "Y"
             steps.append(GooglePlaces(force_refresh=force_execution))
+            steps.append(GooglePlacesDetailed(force_refresh=force_execution))
```

Comment:
Google Place Base info and details will be fetched in the same step.

Reply:
all clear